jerry-core/ecma/operations/ecma-objects-general.c

```diff
@@ -82,7 +82,11 @@ ecma_op_create_object_object_arg (ecma_value_t value) /**< argument of construct
   if (ecma_is_value_object (value)
       || ecma_is_value_number (value)
       || ecma_is_value_prop_name (value)
-      || ecma_is_value_boolean (value))
+      || ecma_is_value_boolean (value)
+#if ENABLED (JERRY_ESNEXT)
+      || ecma_is_value_bigint (value)
```

Comment:
If you move this up, you don't need an extra `)` at the end.

Reply:
good idea!